libs/auth/testing/src/inmemory-backend/auth.service.ts

```diff
@@ -0,0 +1,82 @@
+import { Injectable } from '@angular/core';
+import {
+  InMemoryDbService,
+  STATUS
+} from 'angular-in-memory-web-api';
+import * as faker from 'faker/locale/en_US';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffInMemoryBackendAuthService implements InMemoryDbService {
+  users = {};
+
+  constructor() {}
+
+  private generateToken(): string {
+    return faker.random.alphaNumeric(16);
+  }
+
+  private setUserInfo(email, password) {
+    this.users[email] = {password};
+  }
+
+  createDb(): any {
+    return {
+      auth: this.users
```

Comment:
We don't need to store users. For now, just assume the user exists for logins, and registering returns successfully.